topics/clang.md

```diff
@@ -155,7 +156,7 @@ The linter searches for compile commands in the `build/compile_commands.json` fi
 
 <include from="lib_qd.topic" element-id="root-and-non-root-users-info-bubble"></include>
 
-As mentioned in [prerequisites](#Prepare+your+project), `qodana-clang` reads the `build/compile_commands.json` file and runs the Clang-Tidy tool, and `qodana-cpp` picks up any build system that is supported by CLion and runs analysis on the default build target.
+As mentioned in [prerequisites](#Prepare+your+project), %qdcppc% reads the `build/compile_commands.json` file and runs the Clang-Tidy tool, and non-community %qdcpp% picks up any build system that is supported by CLion and runs analysis on the default build target.
```

Comment:
Do we really have to repeat it once more?
Besides that, the link to the **Prerequisites** section is not valid anymore.

Reply:
Removed

Reply:
Can be resolved now, thank you.